server/aesdsocket.c

```diff
@@ -0,0 +1,216 @@
+#include <stdio.h>
+#include <netdb.h>
+#include <netinet/in.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <unistd.h>
+#include <errno.h>
+#include <syslog.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <signal.h>
+
+#define MAX_BUF_SIZE         (1024)
+#define PORT                 (9000)
+#define SA struct sockaddr   
```

Comment:
@maheshchnt I wouldn't hide the type here with a #define, it makes the code more difficult to read.  A typedef would be better but I'd suggest just referencing struct sockaddr.